packages/tools/src/tools/segmentation/strategies/fillCircle.ts

```diff
@@ -98,9 +95,9 @@ function createPointInEllipse(worldInfo: {
 
   const radius = Math.max(xRadius, yRadius, zRadius);
   if (
-    nearOrZero(xRadius, radius) &&
-    nearOrZero(yRadius, radius) &&
-    nearOrZero(zRadius, radius)
+    csUtils.isEqual(xRadius, radius) &&
```

Comment:
Extract isEqual up on line 19 or so - this function is called enough times that you don't want to dereference the csUtils multiple times.